Dockerfile

```
@@ -0,0 +1,11 @@
+FROM golang:1.16 AS builder
+WORKDIR /go/src/github.com/forbole/big-dipper
+COPY . ./
+RUN CGO_ENABLED=0 go build -o /build/bdjuno ./cmd/bdjuno
+
+FROM alpine:latest
+WORKDIR /root/
```

I think it's always better to move out of the `/root` folder as a workdir, since it might have special rights being owned only by the `root` user:

```suggestion
WORKDIR /bdjuno
```